mautrix_telegram/config.py

```diff
@@ -102,6 +102,12 @@ def do_update(self, helper: ConfigUpdateHelper) -> None:
         copy("metrics.enabled")
         copy("metrics.listen_port")
 
+        copy("telemetry.enabled")
+        copy("telemetry.instance_id")
```

Comment:
Related to my prior comment: if the instance ID is meant to only ever be in a standalone file, it doesn't need to exist as a setting in the config file.